core/src/trezor/wire/thp/cpace.py

```diff
@@ -21,11 +25,28 @@ def generate_keys(self, code_code_entry: bytes) -> None:
         """
         Generate an ephemeral key pair using Elligator2 with X25519.
         """
-        sha_ctx = sha512(_PREFIX)
-        sha_ctx.update(code_code_entry)
-        sha_ctx.update(_PADDING)
-        sha_ctx.update(self.handshake_hash)
-        sha_ctx.update(b"\x00")
+        if __debug__:
```

Comment:
AFAIK assert statements are only active in debug mode anyway, so this seems redundant. https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement